src/coreclr/src/tools/Common/JitInterface/ThunkGenerator/InstructionSetDesc.txt

```diff
@@ -46,18 +47,21 @@ implication        ,X86   ,POPCNT    ,SSE42
 instructionset     ,X86   ,          ,        ,   ,Vector128
 instructionset     ,X86   ,          ,        ,   ,Vector256
 
-; Definition of X64 instruction sets
+; Definition of X64 instruction sets (Define )
```

Comment:
It seems strange to define `instructionset64bit` for a non-64bit architecture.

Reply:
Yeah, custom dsls are weird. But avoiding very large diffs that don't really provide value is worth it.